matrix-sdk-android/src/main/java/org/matrix/android/sdk/internal/session/notification/ProcessEventForPushTask.kt

```diff
@@ -39,14 +40,6 @@ internal class DefaultProcessEventForPushTask @Inject constructor(
 ) : ProcessEventForPushTask {
 
     override suspend fun execute(params: ProcessEventForPushTask.Params) {
-        // Handle left rooms
-        params.syncResponse.leave.keys.forEach {
```

Comment:
these have moved to 

```
roomsJoined = params.syncResponse.join.keys,
roomsLeft = params.syncResponse.leave.keys,
```
https://github.com/vector-im/element-android/pull/4401/files#diff-930446ed1e91a2c4f8a100edef89d21db2bec6b6d4a80765ff58f73e4e042182R88